mne/decoding/tests/test_time_gen.py

```diff
@@ -333,5 +335,11 @@ def test_decoding_time():
     assert_equal("<TimeDecoding | fitted, start : -0.200 (s), stop : 0.499 "
                  "(s), predicted 14 epochs,\n scored (accuracy_score)>",
                  '%s' % tg)
+    # Test with regressor
+    clf = SVR()
+    cv = KFold(len(epochs))
+    y = np.random.rand(len(epochs))
+    tg = TimeDecoding(clf=clf, cv=cv)
```

Comment:
the code should detect automatically that clf is a regressor and switch to KFold. See how it's done in sklearn.

the code should not break when cv is not set and clf is a regressor.